nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-properties-loader/src/main/java/org/apache/nifi/properties/sensitive/SensitivePropertyProvider.java

```diff
@@ -14,10 +14,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.nifi.properties;
+package org.apache.nifi.properties.sensitive;
```

Comment:
Sorry if I missed relevant discussion in previous PRs, but is there any concern about changing the packaging of the classes/interfaces that previously existed?

Reply:
This is definitely a worthwhile question. There are a few structural changes in this PR which I think make things a bit clearer in terms of package layout. We may need to get more eyes on this to decide if it's something that we shouldn't do. For an upcoming follow-up PR, other sensitive property providers will be added and will reference this properties.sensitive package from an external module.

Reply:
I agree that the package layout is clearer (esp. in the context of the other work). And I'm skeptical that this would have been an extension point previously since you'd have to have changed other parts of the codebase, but nonetheless they're public. What are your thoughts on leaving the old stuff in place and deprecating them for a few releases?